auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -214,6 +215,7 @@ FieldSpec generateCreator(ProcessingEnvironment env, List<Property> properties,
 
     MethodSpec.Builder createFromParcel = MethodSpec.methodBuilder("createFromParcel")
         .addAnnotation(Override.class)
+        .addAnnotation(createSuppressUncheckedWarningAnnotation())
```

Comment:
I would really prefer if this was only added when needed instead of on every method.